package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -89,12 +111,18 @@ atom *seg-name*  *residue-number*  *atom-name*
     e.g. ``DMPC 1 C2`` selects the C2 carbon of the first residue of the
     DMPC segment
 
-altloc *alternative-location*
+altLoc *alternative-location*
```

Comment:
note in CHANGELOG (or are we case-insensitive?)

Reply:
We are now; altLoc is the canonical name.